code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -256,18 +256,21 @@
 	var/static/list/names = list("maintenance pill","floorpill","mystery pill","suspicious pill","strange pill")
 	var/static/list/descs = list("Your feeling is telling you no, but...","Drugs are expensive, you can't afford not to eat any pills that you find."\
 	, "Surely, there's no way this could go bad.")
+	prevent_grinding = TRUE
+	dissolvable = FALSE
 
 /obj/item/reagent_containers/pill/floorpill/Initialize()
 	list_reagents = list(get_unrestricted_random_reagent_id() = rand(10,50))
 	. = ..()
 	name = pick(names)
+	desc = "Its surface feels moist and scratched. Your guess is that there is only one way to find out what's inside. "

```

Comment:
Why the hell are you setting the desc in Initialize()?

Reply:
Because the prob(20) one was already there.

Reply:
bruh

Anyways I fixed it